src/components/advancedResources/Card.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import InformationCards from "../cards/InformationCards";
+import HeaderCArd from "../cards/HeaderCArd";
+
+interface Props {
+  Entries: any;
+}
+
+const Card = ({ Entries }: Props) => {
+  const { title, description, entries, colSpan, rowSpan, heightMultiplier } =
+    Entries;
```

Comment:
I don't think in the new structure we need a dedicated card for the advanced resources

Reply:
Yeah! I removed the card from Advanced resources and rename it ResourcesCard in components so that It can be reused because we will need it also on basic resources